docs/Installation.md

```diff
@@ -268,114 +279,118 @@ Note: The Machine Monitor app provides the user with a projection for connected
 
-![machine_monitor](graphics/Machine_Monitor_OH_Counter_4.png)
+![variablesoperation](graphics/variablesoperation1.PNG)
 
 After activating the rule, users can verify the successful creation of the operating hours counter by clicking on the "Operating Hours Counter" rule.
 
-![machine_monitor](graphics/Machine_Monitor_OH_Counter_4_9.png)
+![Activateoperationrule](graphics/Activateoperationrule.PNG)
 
 Users can now confirm that the counter has been successfully created, and the machine's operating hours are being monitored, even without a direct variable tracking the hours.
 
-![machine_monitor](graphics/Machine_Monitor_OH_Counter_5.png)
+![Detailsoperation](graphics/Detailsoperation.PNG)
 
 ### Machine Monitor maintenance logbook and event log
 
 The Machine Monitor app provides the user with two logging mechanisms to track the state of different rules and whether a maintenance job was carried out.
 
 The tab **Maintenance logbook** tracks all the maintenance jobs that have been performed and any changes or resets to the rules. The maintenance job that was carried out earlier in this example is logged in the maintenance logbook.
 
-![machine_monitor](graphics/Machine_Monitor_Maintenance_logbook.png)
+![Maintenancelogbook](graphics/Maintenancelogbook.PNG)
 
 The tab **Event log** provides the user with an overview of the state of the activated rules. It lists details regarding the state of different rules and when they switched states.
 
-![machine_monitor](graphics/Machine_Monitor_Eventlog.png)
+![Eventlog](graphics/Eventlog.PNG)
 
 ### Notification settings
 
 The Machine Monitor apps provides the user with the ability to send notification in case a rule reached the threshold value or a maintenance job is required. The user can either connect the Machine Monitor app to the Notifier app or use the Machine Monitor app directly to send emails.
 
-In order to configure this feature, the user needs to clicks on "Asset configuration".
+In order to configure this feature, the user needs to clicks on "Asset settings".
 
-![machine_monitor](graphics/Machine_Monitor_Asset_Config.png)
+![Assetsettings](graphics/Assetsettings.PNG)
```

Comment:
This first picture is wrong, you meant this one:
![image](https://github.com/industrial-edge/machine-monitor-getting-started/assets/130141248/8559e710-3c6c-4024-9447-782b81738e89)

Reply:
I highlighted this asset settings